docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1385,10 +1366,10 @@ Remember because assertions are used to describe a condition of the previous
 commands - the `timeout` modification goes on the previous commands _not the
 assertions_.
 
-#### Example #1: Default Assertion
+#### Example #1: Implicit Assertion
```

Comment:
Above the section:
```
### Writing Assertions
There are two ways to write assertions in Cypress:
1. **Implicit Subjects:** Using [`.should()`](/api/commands/should) or
   [`.and()`](/api/commands/and).
2. **Explicit Subjects:** Using `expect`.
```

why is .should()/.and() an implicit subject?  it'd just be a queued assertion vs sync assertion right?

Reply:
Yeah, I think that section could use a touch-up. Rewriting it.

Reply:
since https://github.com/cypress-io/cypress/pull/25738 is no longer making its way into the v13 release, I think this suggestion is no longer applicable.